cmd/create_config.go

```
@@ -82,18 +82,18 @@ func init() {
 
 			v, err := config.CreateConfigFile(data)
 			if err != nil {
-				panic(err)
+				log.Fatalf("error: %v", err)
 			}
 			log.Infof("Config File Created: %s\n", v.ConfigFileUsed())
 
-			ctx := baseBootstrap(v.ConfigFileUsed())
+			ctx, canc, _ := commandBootstrap(v.ConfigFileUsed())
```

let us not ignore the err here ?